tests/baselines/reference/controlFlowAnalysisOnBareThisKeyword.types

```diff
@@ -0,0 +1,39 @@
+=== tests/cases/compiler/controlFlowAnalysisOnBareThisKeyword.ts ===
+declare function isBig(x: any): x is { big: true };

```

Comment:
Shouldn’t there be way more type differences than this, since we are now narrowing `this` in classes? Maybe I’m missing something about the actual change.

Reply:
[Later] Yes, I did. But github doesn’t display the “Are you sure you want to delete your comment?” message on the iPad, so I can’t delete it.